### DISCUSSION TITLE
Build hardware

### DISCUSSION THREAD

**Discussion by btwbtw01** [Xilinx/finn#558] [Q&A]
Hi, I have trained VGG16 model with Cifar10 dataset. I want to build hardware and deploy on ZCU104. It shows the error.
![VGG16_error](https://user-images.githubusercontent.com/85877169/155143401-ee456763-c868-42cc-9c09-766c81918d34.PNG)
![VGG16_error_detail](https://user-images.githubusercontent.com/85877169/155143426-7a7b8a68-571c-4c63-b461-45626ecfb968.PNG)
How can I solve this error? Thanks!!

**Comment by fpjentzsch** [comment]
Hi, why does your terminal prompt state "I have no name!" as the username? There seems to be a bigger underlying issue here, such as memory corruption or a permissions issue with your Ubuntu installation.

**Reply by btwbtw01** [reply]
Hi, @fpjentzsch 
I'm using server to train model and compile model. I have no root permission, just normal user. But I can run the example without any errors(except tfc_end2end_example.ipynb can't run DeployToPYNQ). 
By the way, the hls model will generate at /tmp folder. Can I change the folder to put hls model?
